src/dbus_api.rs

```diff
@@ -241,6 +297,13 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
             let dbus_context_clone = dbus_context.clone();
             let object_path: dbus::Path = create_dbus_pool(dbus_context_clone);
             let (rc, rs) = code_to_message_items(StratisErrorEnum::STRATIS_OK);
+
+            // Insert a mapping of the object path name to the pool name to be
+            // used in the pool interface methods to look up the Stratis pool.
```

Comment:
This cloning really worries me, because I know it means we're making a copy every time. There seem to be issues of efficiency and correctness. Other than that, I definitely think we need this table, somehow, so this is right.